depict/utils.py

```diff
@@ -118,10 +118,15 @@ def create_svg(m):
 
 def create_png_jpeg_image(m, filename, format, size, smarts):
     # extension = ImageFormat.PNG.value if format == ImageFormat.PNG.value else ImageFormat.JPG.value
-    smart = Chem.MolFromSmarts(smarts)
-    highlight = m.GetSubstructMatch(smart)
-    print(highlight)
-    pil_image = Draw.MolToImage(m, size= size, highlightAtoms = highlight)
+    substructure = Chem.MolFromSmarts(smarts)
+    all_atom_matches = m.GetSubstructMatches(substructure)
+    bond_matches = []
+    for atom_matches in all_atom_matches:
+        for bond in substructure.GetBonds():
+            idx1, idx2 = bond.GetBeginAtomIdx(), bond.GetEndAtomIdx()
+            bond_matches.append(m.GetBondBetweenAtoms(atom_matches[idx1], atom_matches[idx2]).GetIdx())
+    all_atom_matches = sum(m.GetSubstructMatches(substructure), ()) # this just combines the tuple of tuples into a single tuple
```

Comment:
@Jack-42 `all_atom_matches` value does not change here
Can this line be changed to `all_atom_matches = sum(all_atom_matches, ())`?

I have tested the rest, it is working fine

Thanks for making the change

Reply:
Yes you're right! Thanks for catching that. The line change has been made in the most recent commit